File: app/src/main/java/com/imagine/mohamedtaha/store/ItemsStore.java
package com.imagine.mohamedtaha.store;

/**
 * Created by MANASATT on 03/12/17.
 */

public class ItemsStore {
    public int getId() {
        return id;
    }

    public void setId(int id) {
        this.id = id;
    }

    int id;
    String typeStore;

    public String getTypeStore() {
        return typeStore;
    }

    public void setTypeStore(String typeStore) {
        this.typeStore = typeStore;
    }

    public String getNotes() {
        return notes;
    }

    public void setNotes(String notes) {
        this.notes = notes;
    }

    public String getCreatedDate() {
        return createdDate;
    }

    public void setCreatedDate(String createdDate) {
        this.createdDate = createdDate;
    }

    String notes;
    String createdDate;
}

File: app/src/main/java/com/imagine/mohamedtaha/store/data/ItemsStore.kt
package com.imagine.mohamedtaha.store.data

import androidx.room.ColumnInfo
import androidx.room.Entity
import androidx.room.PrimaryKey

/**
 * Created by MANASATT on 03/12/17.
 */
public class ItemsStore {
    var id:Int?= null
    var namePermission:String? = null
    var typeStore: String? = null
    var notes: String? = null
    var createdDate: String? = null
    var createdTime: String? = null
    var convertTo: String? = null
    var nameGategory: String? = null
    var nauralCategory: String? = null
    var userName: String? = null
    fun getmTimeInMilliseconds(): Long {
        return mTimeInMilliseconds
    }

    fun setmTimeInMilliseconds(mTimeInMilliseconds: Long) {
        this.mTimeInMilliseconds = mTimeInMilliseconds
    }

    private var mTimeInMilliseconds: Long = 0
    var issued = 0
    var incoming = 0
    var id_code_category: Long = 0
    var id_code_store: Long = 0
    var id_permission_id: Long = 0
    var id_convert_to: Long = 0
    var userId: Long = 0
    var first_balanse = 0
}